web_src/js/features/common-organization.js

```diff
@@ -1,14 +1,11 @@
-import $ from 'jquery';
 import {initCompLabelEdit} from './comp/LabelEdit.js';
 import {toggleElem} from '../utils/dom.js';
 
 export function initCommonOrganization() {
-  if ($('.organization').length === 0) {
-    return;
-  }
-
-  $('.organization.settings.options #org_name').on('input', function () {
-    const nameChanged = $(this).val().toLowerCase() !== $(this).attr('data-org-name').toLowerCase();
+  const orgNameInput = document.querySelector('.organization.settings.options #org_name');
+  if (!orgNameInput) return;
```

Comment:
Oh wait .... it seem incorrect .....

There is another call `initCompLabelEdit` below. It shouldn't return here.

So the old logic (like `  if ($('.organization').length === 0) {` ) is still needed.

Reply:
Oh right, reverting.